lib/karafka/schemas/config.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module Karafka
+  # Namespace for all the validation schemas that we use to check input
+  module Schemas
+    # Regexp for validating format of groups and topics
```

Comment:
Maybe add some examples of correct namings? for those who are not fluent in regexp it can be helpful.